pkg/api/rest/types.go

```diff
@@ -60,7 +81,7 @@ type podStrategy struct {
 
 // Pods is the default logic that applies when creating and updating Pod
 // objects.
-var Pods RESTCreateStrategy = podStrategy{api.Scheme, api.SimpleNameGenerator}
+var Pods = podStrategy{api.Scheme, api.SimpleNameGenerator}
```

Comment:
Also, aren't these included in RESTCreateStrategy directly? Should RESTCreateStrategy use this strategy? Does RESTUpdateStrategy invoke RESTCreateStrategy for the create-on-put case? 

I'm not getting the big picture of the organization we're moving towards.

Don't get me wrong -- this is better than what we have. But, I'm still not seeing the forest through the trees.